components/elm/src/biogeophys/EnergyFluxType.F90

```diff
@@ -380,8 +392,171 @@ subroutine Restart(this, bounds, ncid, flag)
          long_name='', units='', &
          interpinic_flag='interp', readvar=readvar, data=this%btran2_patch) 
 
+    call restartvar(ncid=ncid, flag=flag, varname='BTRAN_MIN', xtype=ncd_double,  &
+         dim1name='pft', &
+         long_name='daily minimum of transpiration wetness factor', units='', &
+         interpinic_flag='interp', readvar=readvar, data=this%btran_min_patch) 
+
+    call restartvar(ncid=ncid, flag=flag, varname='BTRAN_MIN_INST', xtype=ncd_double,  &
+         dim1name='pft', &
+         long_name='instantaneous daily minimum of transpiration wetness factor', units='', &
+         interpinic_flag='interp', readvar=readvar, data=this%btran_min_inst_patch) 
+
```

Comment:
Why does this instantaneous value need to be written out in the restart file? Won't this value be calculated on the fly and be available for use?

Reply:
I checked the code again, and my understanding is that the code uses variable `btran_min_inst` as the placeholder for the minimum during the day, then it transfers the result to `btran_min` once the day finishes. So if the restart file is from the middle of the day, it needs to retrieve `btran_min_inst` to continue the check for the true minimum of the day split between before and after the restart.